takwimu/templates/takwimu/_includes/navbar/_index.html

```diff
@@ -99,3 +100,17 @@
     </div>
   </div>
 </nav>
+
+<style>
+  .navbar.fixed-top .dropdown-menu-lg li[type="disc"] {
```

Comment:
Why the `type` argument? And why `"disc"`? I suppose we want to apply the same styling to all of the menu links?

Reply:
`Type` is specific to all menu links in the drop-down menu. Targeting `type` excludes the social media icon list